<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabclient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1084</width>
      <height>506</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Ajout</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Id:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_id">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Nom:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_nom"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Prenom:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEdit_prenom"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Telephone:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="lineEdit_tel"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Adresse:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="lineEdit_adresse"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Email:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="lineEdit_email"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pb_ajouter">
               <property name="text">
                <string>Ajouter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="resetClient">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Affichage</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEditrech"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pb_rechercher">
             <property name="text">
              <string>Rechercher</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reaficherClient">
             <property name="text">
              <string>Reaficher</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tabclient_2"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Statistique</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_tri">
             <property name="text">
              <string>Tri</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pb_modifier">
             <property name="text">
              <string>Modifier</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_supprimer">
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Commande</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Ajout</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>id:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_id2"/>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Nom:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_nom2"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Prix:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_prix"/>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_type"/>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>CinP:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_cinp"/>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pb_ajouter2">
               <property name="text">
                <string>Ajouter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="resetCommande">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Affichage</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_17">
             <item>
              <widget class="QLineEdit" name="lineEditrech2"/>
             </item>
             <item>
              <widget class="QPushButton" name="pb_rechercher2">
               <property name="text">
                <string>Rechercher</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reaficherCommande">
               <property name="text">
                <string>Reaficher</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="tabcommande_2"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="stat_Commande">
          <property name="text">
           <string>Statistique</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_tri2">
          <property name="text">
           <string>Tri</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_modifier2">
          <property name="text">
           <string>Modifier</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_supprimer2">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_16"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
